packages/elements-core/src/components/Docs/HttpOperation/HttpOperation.spec.tsx

```diff
@@ -70,6 +70,51 @@ describe('HttpOperation', () => {
       unmount();
     });
 
+    it('displays scopes in badge when present', () => {
+      const security = [
+        [
+          {
+            key: 'oauth2WithScopes',
+            type: 'oauth2',
+            description: 'foo',
+            flows: {
+              implicit: {
+                scopes: {
+                  'write:pets': 'modify pets in your account',
+                  'read:pets': 'read your pets',
+                },
+                refreshUrl: 'http://refreshUrl.com',
+                authorizationUrl: 'http://authorizationUrl.com',
+              },
+            },
+          },
+          {
+            key: 'oauth2WithEmptyScopes',
+            type: 'oauth2',
+            description: 'foo',
+            flows: {
+              authorizationCode: {
+                scopes: {},
+                refreshUrl: 'http://refreshUrl.com',
+                tokenUrl: 'http://tokenUrl.com',
+                authorizationUrl: 'http://authorizationUrl.com',
+              },
+            },
+          },
+        ],
+      ];
+
+      const { unmount } = render(<HttpOperation data={{ ...httpOperation, security: security }} />);
```

Comment:
Getting the following and im trying to figure out whats wrong with `security`.

> Type '({ key: string; type: string; description: string; flows: { implicit: { scopes: { 'write:pets': string; 'read:pets': string; }; refreshUrl: string; authorizationUrl: string; }; authorizationCode?: undefined; }; } | { ...; })[][]' is not assignable to type 'HttpSecurityScheme[][]'.

Reply:
It's because in `type: 'oauth2'` `oauth2` is treated as string, not a literal as in a type definition. Making it `type: 'string' as const` fixes the issue.